.github/workflows/ci.yml

```diff
@@ -81,7 +79,7 @@ jobs:
         # https://doc.rust-lang.org/cargo/commands/cargo-test.html#target-selection
         run: |
           echo "Running command: cargo test $RELEASE_OPTION $ALL_FEATURES_OPTION"
-          cargo test $RELEASE_OPTION $ALL_FEATURES_OPTION
+          cargo test $RELEASE_OPTION $ALL_FEATURES_OPTION -- --include-ignored
```

Comment:
This will always include the ignored tests. Maybe we can move this a few lines up inside the $RELEASE_OPTION ? https://github.com/tlsnotary/tlsn/issues/500

I'd also document this. suggestion: ```
        # `-- --include-ignored` to include the ignored (expensive integration) tests
```